dealers_choice.js

```diff
@@ -11,26 +11,313 @@ app.use(staticMiddleWare);
 
 app.get("/", (request, response) => {
   const entries = postBank.list();
-  const html = `
+  const filteredDificultyPeaks = postBank.getAllPeaksFromDifficulty;
+  let randomIdx = () => Math.floor(Math.random() * entries.length);
+  const homeHtml = `
   <!DOCTYPE html>
   <html lang="en">
     <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>The Fourteeners</title>
+      <meta charset="UTF-8">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>The Fourteeners</title>
+      <link rel="stylesheet" href="/style.css" />
+      <script src="https://kit.fontawesome.com/dc998fff98.js" crossorigin="anonymous"></script>
+      <link rel="preconnect" href="https://fonts.googleapis.com">
+      <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+      <link href="https://fonts.googleapis.com/css2?family=Permanent+Marker&display=swap" rel="stylesheet">
     </head>
```

Comment:
since you're using this same block of code in all of your routes, maybe you can pull it out to a separate `const` (or even a separate file) and just import it in here
1. make a new file called htmlSnippets.js
in htmlSnippets.js
```js
module.exports = {
   header: `
    <!DOCTYPE html>
    <html lang="en">
      <head>
        <meta charset="UTF-8">
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <meta name="viewport" content="width=device-width, initial-scale=1.0">
      <title>The Fourteeners</title>
      <link rel="stylesheet" href="/style.css" />
      <script src="https://kit.fontawesome.com/dc998fff98.js" crossorigin="anonymous"></script>
      <link rel="preconnect" href="https://fonts.googleapis.com">
      <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
      <link href="https://fonts.googleapis.com/css2?family=Permanent+Marker&display=swap" rel="stylesheet">
    </head>
  `,
};
```
2. in dealers_choice.js:
```js
const { header } = require('./htmlSnippets');

...
const homeHTML = ``
  ${header}
   ...the rest of your code
;
```

you can do this with ANY code snippet that you end up reusing a lot